app/code/local/Magmodules/Webwinkelconnect/Helper/Data.php

```diff
@@ -77,4 +77,53 @@ public function getShopId() {
         return Mage::getStoreConfig('webwinkelconnect/general/api_id');
     }
 
+    public function hasCredentialFields(?string $webshop_id, ?string $api_key): bool {
+        return isset($webshop_id) && isset($api_key);
+    }
+
+    public function credentialsEmpty(?string $webshop_id, ?string $api_key): bool {
+        return !trim($webshop_id) || !trim($api_key);
+    }
+
+    public function isAuthorized(string $webshop_id, string $api_key): void {
+        if ($webshop_id == $this->getShopId() && hash_equals($api_key, $this->getApiKey())) {
+            return;
+        }
+        $this->returnResponseCode(401, 'Wrong credentials');
+    }
+
+    public function syncProductReview(array $product_review): void {
+        $review = Mage::getModel('review/review')
+            ->setEntityPkValue($product_review['product_review']['product_id'])
+            ->setStatusId(Mage_Review_Model_Review::STATUS_PENDING)
+            ->setTitle($product_review['product_review']['title'])
+            ->setDetail(str_replace('<br>', "\n", $product_review['product_review']['review']))
```

Comment:
Do we send the reviews from the dashboard as HTML? Is `<br>` the only tag that can appear? What happens with other tags?

Reply:
@to6ko12 I see that you've removed the sanitization, however, we do allow HTML tags in our reviews.
You have to check if Magento espaces the HTML when it loads the reviews. If it doesn't then you have to sanitize the review, check  https://www.php.net/manual/en/function.htmlentities.php